assets/i18n/pt.json

```diff
@@ -252,6 +252,7 @@
   "I18N_SIGNUP_WAIVER_OBJECTIVE":"A renúncia do requisito de atribuição (BY) significa que, se alguém reutilizar um trabalho, não é exigido que sejam indicados os autores. Contudo, todas as contribuições individuais para explorações ficarão disponíveis neste site, no registo de alterações, e as pessoas que reutilizarem a exploração serão encorajadas, mas não obrigadas a incluir o link desta página.",
   "I18N_SIGNUP_WHY_LICENSE": "Porquê a licença CC-BY-SA?",
   "I18N_SPLASH_FIRST_EXPLORATION_DESCRIPTION": "As explorações são aulas interactivas que proporcionam uma experiência mais imersiva do que videos e textos estáticos, ajudando e os utilizadores aprendem fazendo.",
+  "I18N_SPLASH_FIRST_EXPLORATION_DESCRIPTION_ALTERNATE": "Aulas de Oppia, também conhecidos como explorações, proporcionam uma experiência mais imersiva do que videos e textos estáticos, ajudando e os utilizadores aprendem fazendo.",
```

Comment:
Olá, bom dia.

The sentences are correct, except the last one. Here it goes:

" > +  "I18N_SPLASH_FIRST_EXPLORATION_DESCRIPTION_ALTERNATE": "Aulas de
Oppia, também *conhecidas* como explorações, proporcionam uma experiência
mais imersiva do que videos e textos estáticos, ajudando e os utilizadores
aprendem fazendo.",

On 31 Jul 2016 08:58, "Sean Lip" <notifications@github.com> wrote:

> In assets/i18n/pt.json
> <https://github.com/oppia/oppia/pull/2313#discussion_r72900122>:
>
> > @@ -252,6 +252,7 @@
> >    "I18N_SIGNUP_WAIVER_OBJECTIVE":"A renúncia do requisito de atribuição (BY) significa que, se alguém reutilizar um trabalho, não é exigido que sejam indicados os autores. Contudo, todas as contribuições individuais para explorações ficarão disponíveis neste site, no registo de alterações, e as pessoas que reutilizarem a exploração serão encorajadas, mas não obrigadas a incluir o link desta página.",
> >    "I18N_SIGNUP_WHY_LICENSE": "Porquê a licença CC-BY-SA?",
> >    "I18N_SPLASH_FIRST_EXPLORATION_DESCRIPTION": "As explorações são aulas interactivas que proporcionam uma experiência mais imersiva do que videos e textos estáticos, ajudando e os utilizadores aprendem fazendo.",
> > +  "I18N_SPLASH_FIRST_EXPLORATION_DESCRIPTION_ALTERNATE": "Aulas de Oppia, também conhecidos como explorações, proporcionam uma experiência mais imersiva do que videos e textos estáticos, ajudando e os utilizadores aprendem fazendo.",
>
> Hi @anafrancisca <https://github.com/anafrancisca>, I added this line. Is
> it correct? Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/oppia/oppia/pull/2313/files/5db567d0cf4a7f9f1dbb32f2f95470a7cfbbc84b#r72900122>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARewAvbFwdm7AmUqK6cITd2tMzRinzDUks5qbFWggaJpZM4JY-4r>
> .
>